src/topology/path_connected.lean

```diff
@@ -138,6 +138,26 @@ begin
   simp
 end
 
+open continuous_map
+
+instance : has_coe (path x y) C(I, X) := ⟨λ γ, γ.1⟩
+
+instance : topological_space (path x y) :=
```

Comment:
The topology is standard but maybe you want to add a docstring to describe what it is?

Reply:
Like this?

Reply:
Looks good!